src/comm/LinkManager.cc

```diff
@@ -481,7 +481,10 @@ void LinkManager::_updateConfigurationList(void)
                 }
             } else {
                 // Lets create a new Serial configuration automatically
-                pSerial = new SerialConfiguration(QString("Pixhawk on %1").arg(portInfo.portName().trimmed()));
+		if (portInfo.description() == "AeroCore")
```

Comment:
Why not use the portInfo.description() for the name of the device?  The Pixhawk has "PX4 v2 ..." or something like that but could just be parsed out.